charts/kong/README.md

```diff
@@ -466,8 +470,8 @@ The exact plugin settings may vary in your environment. The `secret` should
 always be changed for both configurations.
 
 After creating your secret, set its name in values.yaml, in the
-`.enterprise.rbac.session_conf_secret` and
-`.enterprise.portal.session_conf_secret` keys.
+`.enterprise.rbac.session_conf_secret` and optionally in
+`env.portal_session_conf` using a secretKeyRef.
```

Comment:
To promote good security practices, let's encourage secretKeyRef rather than mentioning it as an option.